sdk/core/Azure.Core/src/JsonPatchDocument.cs

```diff
@@ -126,28 +146,53 @@ public void AppendTest<T>(string path, T value)
         }
 
         /// <summary>
-        /// Returns a formatted JSON string representation of this <see cref="JsonPatchDocument"/>.
+        /// Returns a UTF8 encoded representation of this <see cref="JsonPatchDocument"/> instance.
```

Comment:
Nit:
```suggestion
        /// Returns a UTF8-encoded representation of this <see cref="JsonPatchDocument"/> instance.
```

Reply:
A few others below. I'll not repeat. Basically, it's a compound adjective and should be hyphenated, but not a big deal (i.e. nit).